src/edge_13-79/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bugs": "https://github.com/alrra/browser-logos/issues",
+  "description": "Microsoft Edge v13-79 browser logo",
```

Comment:
If I remember correctly the versions were: `12-18`.

Cc: @kypflug.

Reply:
12-18 is correct for EdgeHTML, but we didn't actually change the logo until 79, so putting this as 12-78 would probably be most accurate.

Reply:
@kypflug, thanks for the info! I’ll update the PR.

I presumed it was 12-79 because the icon is changing while Edge Canary is 80. And, well, 80 is a clean base 10 number. 😃

Reply:
And done. Please let me know if there is anything else I can do. Odd as it may seem, I’m excited to finally contribute to this repository in a tangible way.

Reply:
> 12-18 is correct for EdgeHTML, but we didn't actually change the logo until 79, so putting this as 12-78 would probably be most accurate.

@kypflug, @jonathantneal Even with the jump, I think we should use `12-18` to reflect reality (i.e. there was never something like a version 50). 

Ref: https://developer.microsoft.com/en-us/microsoft-edge/platform/changelog/

![](https://user-images.githubusercontent.com/1223565/68161275-60c83480-ff0a-11e9-9433-2f6160517145.png)

Reply:
Sounds good to me. Thanks @alrra!

Reply:
@alrra, you got it. The change has been made!